.gitignore

```diff
@@ -44,3 +44,5 @@
 .dockerdev/*
 docker-compose.yml
 dip.yml
+
+.idea/
```

Comment:
Can you add a comment like line 43 that explains what this ignore is for?

Reply:
Done!